Source/Core/DolphinQt/Utils/Resources.cpp

```diff
@@ -70,10 +70,8 @@ void Resources::UpdatePixmaps()
 	m_pixmaps[TOOLBAR_FULLSCREEN].load(GIFN("fullscreen"));
 	m_pixmaps[TOOLBAR_SCREENSHOT].load(GIFN("screenshot"));
 	m_pixmaps[TOOLBAR_CONFIGURE].load(GIFN("config"));
-	m_pixmaps[TOOLBAR_PLUGIN_GFX].load(GIFN("graphics"));
-	m_pixmaps[TOOLBAR_PLUGIN_DSP].load(GIFN("dsp"));
-	m_pixmaps[TOOLBAR_PLUGIN_GCPAD].load(GIFN("gcpad"));
```

Comment:
Why are we removing these? Don't the files still exist? I plan to use them for the Config dialogue.